build.sh

```diff
@@ -321,37 +323,29 @@ build_jdk() {
     chmod -R 755 ${wrkdir}/openjdk/build || exit $?
 }
 
-MX_REPO=https://bitbucket.org/allr/mx
-MX_VERSION=072da2b
-GRAAL_REPO=http://hg.openjdk.java.net/graal/graal-compiler
-GRAAL_VERSION=graal-0.13
-# Building with the JDK we built earlier is troublesome (SSL+maven issues).
-# Instead we use the system JDK8.
-case `uname` in
-    Linux)   SYSTEM_JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-amd64;;
-    OpenBSD) SYSTEM_JAVA_HOME=/usr/local/jdk-1.8.0;;
-    *) unknown_platform;;
-esac
-MX="env DEFAULT_VM=jvmci JAVA_HOME=${SYSTEM_JAVA_HOME} python2.7 ${wrkdir}/mx/mx.py"
+# The latest Graal and MX at the time of writing.
+# MX doesn't have releases.
+MX_REPO=https://github.com/graalvm/mx
```

Comment:
Elsewhere in the file, we have the "static" things (like websites) in the body of the functions, and only the "changeable" things (version numbers) as variables.